pulser-core/pulser/channels/dmm.py

```diff
@@ -72,26 +89,50 @@ def basis(self) -> Literal["ground-rydberg"]:
     def _undefined_fields(self) -> list[str]:
         optional = [
             "bottom_detuning",
+            "total_bottom_detuning",
```

Comment:
What do you think of allowing `total_bottom_detuning` do be left as `None` in physical devices by modifying `DMM.is_virtual()`? I would be in favor of doing so temporarily and raising a `DeprecationWarning` telling the user to define it.  This would allow us to keep backwards compatibility from 0.15 to 0.16. 
It would also involve not defining it for now in Chadoq2 and making the serializer skip this parameter when it is not `None`.
What do you think?

Reply:
Yes, I am in favor of this (especially because we saw nasty errors in the past due to changes in the serialization).